cliquet/resource/__init__.py

```diff
@@ -611,6 +611,11 @@ def _add_cache_header(self, response):
         collection_expires = self.request.registry.settings.get(setting_key)
         if collection_expires is not None:
             response.cache_expires(seconds=int(collection_expires))
+        else:
+            # Since Expires has resolution in seconds, do not use cache_expires
+            # in order to omit it.
+            response.cache_control.no_cache = True
+            response.pragma = 'no-cache'
```

Comment:
I don't think `pragma` is a response header, even in HTTP/1.0 it was only a request header, right? See http://stackoverflow.com/a/15050018/680454